pravega-cli/src/main/resources/config.properties

```diff
@@ -10,20 +10,24 @@
 
 # Config for CLI credentials (at the moment, Controller auth is only supported).
 cli.authEnabled=false
+cli.tlsEnabled=false
 cli.userName=admin
 cli.password=1111_aaaa
 
+# Config for CLI security
+cli.security.tls.trustStore.location=./conf/client.truststore.jks
+
 # Config for CLI Controller REST/GRPC endpoints.
-cli.controllerRestUri=http://localhost:10080
-cli.controllerGrpcUri=tcp://localhost:9090
+cli.controllerRestUri=localhost:9091
+cli.controllerGrpcUri=localhost:9090
 cli.metadataBackend=segmentstore
 
 # General configuration for the Pravega cluster at hand.
 pravegaservice.clusterName=pravega
```

Comment:
This and others in this file are old config names. Please use the new names. See the mapping of old to new names here: https://github.com/pravega/pravega/wiki/Configuration-Name-Changes.

Reply:
Maybe this can be done on a separate PR, right? Just to do not mix the security stuff with the naming convention work.

Reply:
That sounds good. You are referring to the Pravega config as well as the existing CLI config right? For any new Pravega tools config that we introduce, IMO we should start using new config guidelines. What do you say?

Reply:
Yeah changes regarding the new config guidelines will be made.